plugins/plugin-activity/che-plugin-activity-server/pom.xml

```diff
@@ -11,17 +11,23 @@
       Codenvy, S.A. - initial API and implementation
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
It seems that we have numerous usages of both URLs in different POMs throughout entire Che project for some reason.
And this happened because I was creating this pom by copying from another, which has this different url. I can revert it, but it seems it doesn't matter much, as _http://maven.apache.org/maven-v4_0_0.xsd_ simply redirects to http://maven.apache.org/xsd/maven-4.0.0.xsd 

Then one can normalize all poms to use same xsd URL across the project later (as a separate task) if needed.